tests/python/test_random.py

```diff
@@ -128,7 +124,7 @@ def gen():
     assert not np.allclose(result[0], result[1])
 
 
-@ti.test(arch=[ti.cpu, ti.cuda])
+@ti.test(arch=[ti.cpu, ti.cuda, ti.metal])
```

Comment:
Our CI's Metal doesn't support 64-bit data yet

```suggestion
@ti.test(arch=[ti.cpu, ti.cuda])
```

Reply:
Got it!